vizro-core/tests/unit/vizro/models/test_models_utils.py

```diff
@@ -41,3 +48,30 @@ def test_check_for_valid_component_types(self, model_with_layout):
             ),
         ):
             model_with_layout(title="Page Title", components=[vm.Checklist()])
+
+
+class TestWarnDescriptionWithoutTitle:
+    @pytest.mark.parametrize(
+        "model_type",
+        [
+            "dashboard",
+            "page",
+            "container",
+            "tabs",
+            "graph",
+            "table",
+            "ag_grid",
+            "dropdown",
+            "radio_items",
+            "checklist",
+            "slider",
+            "range_slider",
+            "date_picker",
+            "user_input",
+            "text_area",
+        ],
+    )
+    def test_warns_if_description_and_no_title(self, model_type):
+        info = MockValidationInfo(data={"title": "", "type": model_type})
+        with pytest.warns(UserWarning, match="description.*title.*missing or empty"):
+            warn_description_without_title("description", info)
```

Comment:
The `model_type` is not relevant anymore. 

```suggestion
class TestWarnDescriptionWithoutTitle:
    def test_warns_if_description_and_no_title(self):
        info = MockValidationInfo(data={"title": ""})
        with pytest.warns(UserWarning, match="description.*title.*missing or empty"):
            warn_description_without_title("description", info)
```

Also if you finds it suitable, move the test under the `TestSharedValidators` and skip using the new `TestWarnDescriptionWithoutTitle` just for this test.